redi-connect-front/src/components/organisms/Avatar.tsx

```diff
@@ -52,23 +52,24 @@ const Avatar = ({ profile, profileSaveStart }: any) => {
   }
 
   return (
-    <div className={classnames('avatar', {
+    <div className={classnames('avatar', `avatar--border${mePage ? '__me' : '__other'}`, {
```

Comment:
I have a simple rule. If I find my self adding too much "if"s I need to start over. 
From 5 changes I see 4x are introducing ifs. Please re-thing the solution

Reply:
solved due to new component